test/fixtures/dependencies.js

```diff
@@ -2,7 +2,7 @@ const fs = require('fs');
 const path = require('path');
 const android = require('./android');
 
-const pjson = fs.readFileSync(path.join(__dirname, './files/dependency.json'));
+const pjson = fs.readFileSync(path.join(__dirname, 'files', 'package.json'));
```

Comment:
But you are relying on the package.json that was made for dependency tests, not for plugin system. And that's bad stuff :( 

(I am not talking about relying on `node_modules` dependencies)